test/server/cards/02_SHD/events/CalculatedLethality.spec.ts

```diff
@@ -0,0 +1,74 @@
+describe('Calculated Lethality', function () {
+    integration(function (contextRef) {
+        describe('Calculated Lethality\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
```

Comment:
Might as well move the setup into the test case since there's only one case. Same for other test files